File-Formats/DocIO/Applying-Watermark.md

```diff
@@ -168,7 +105,11 @@ The following code example illustrates how to add a picture watermark to the Wor
 
 {% tabs %}  
 
-{% highlight c# tabtitle="C#" %}
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+//DocIO supports picture watermark in Windows Forms, WPF, ASP.NET and ASP.NET MVC platforms alone.
```

Comment:
We supported picuture watermark in cross platforms, please refer the sample here
https://github.com/SyncfusionExamples/DocIO-Examples/blob/main/Watermark/Add-picture-watermark/.NET-Standard/Add-picture-watermark/Program.cs


So, add the proper code example